example/session/src/main/java/org/apache/iotdb/AlignedTimeseriesSessionExample.java

```diff
@@ -573,9 +573,9 @@ private static void insertTabletsWithAlignedTimeseries()
       tablet3.addTimestamp(row3, timestamp);
       for (int i = 0; i < 2; i++) {
         long value = new SecureRandom().nextLong();
-        tablet1.addValue(schemaList1.get(0).getSubMeasurementsList().get(i), row1, value);
-        tablet2.addValue(schemaList2.get(0).getSubMeasurementsList().get(i), row2, value);
-        tablet3.addValue(schemaList3.get(0).getSubMeasurementsList().get(i), row3, value);
+        tablet1.addValue(schemaList1.get(0).getMeasurementId(), row1, value);
+        tablet2.addValue(schemaList2.get(0).getMeasurementId(), row2, value);
+        tablet3.addValue(schemaList3.get(0).getMeasurementId(), row3, value);
```

Comment:
The correct code should be this?
```suggestion
        tablet1.addValue(schemaList1.get(i).getMeasurementId(), row1, value);
        tablet2.addValue(schemaList2.get(i).getMeasurementId(), row2, value);
        tablet3.addValue(schemaList3.get(i).getMeasurementId(), row3, value);
```